Refactor/order/#199

Joowon-Seo opened the pull request:
## 개요
issue : #199 

## PR 유형
어떤 변경 사항이 있나요?

- [x] 코드 리팩토링
- [x] 주석 추가 및 수정
- [x] 테스트 추가, 테스트 리팩토링

> ### Description

- 페이징 메소드 리팩토링
- 장바구니 추가 객체지향 코드로 변경
- Event FallBack = true 제거

Joowon-Seo committed:
refactor : 주문 조회 페이징 수정

Joowon-Seo committed:
refactor : 주문상세 페이징 수정

Joowon-Seo committed:
refactor : 장바구니 추가 리펙토링
객체지향적 코드 작성

Joowon-Seo committed:
refactor : 필요없는 event option 제거
FallBack = true 제거

Joowon-Seo committed:
Fix : Qodana build test fail

Joowon-Seo committed:
Fix : qodana.yaml 수정
Qodana Scan fail

dodari79 commented on order-api/src/main/java/org/ecommerce/orderapi/bucket/service/BucketDomainService.java:
```diff
@@ -52,23 +52,19 @@ public BucketDto addBucket(
 
-		validateStock(request.productId(), request.quantity());
+		Integer quantity = bucket.getQuantity() + request.quantity();
+		bucket.appendQuantity(quantity);
```
appendQuiantity메소드명은 quantity추가하는 이름 같은데 추가된 값을 던지는게 아니라 추가할 requestr의 quantity만 던지면 되지 않을까 싶습니다.

Joowon-Seo committed:
refactor : 장바구니 추가 수정
appendQuantity 파라미터 값 수정
#본문: 코드 의도 설명 가능한 자세하게

# Feat     : 새로운 기능 추가
# Fix      : 버그 수정
# Docs     : 문서 수정
# Test     : 테스트 코드 추가
# Refactor : 코드 리팩토링, 파일 혹은 폴더명을 수정하거나 옮기는 작업만인 경우, 파일을 삭제하는 작업만 수행한 경우
# Design   : CSS 및 레이아웃 작업
# Chore    : 코드 외 빌드 부분 혹은 패키지 매니저 수정사항

Joowon-Seo committed:
refactor : StockDomainService 수정
stockerOperationResult method 사용
#본문: 코드 의도 설명 가능한 자세하게

# Feat     : 새로운 기능 추가
# Fix      : 버그 수정
# Docs     : 문서 수정
# Test     : 테스트 코드 추가
# Refactor : 코드 리팩토링, 파일 혹은 폴더명을 수정하거나 옮기는 작업만인 경우, 파일을 삭제하는 작업만 수행한 경우
# Design   : CSS 및 레이아웃 작업
# Chore    : 코드 외 빌드 부분 혹은 패키지 매니저 수정사항

Joowon-Seo committed:
refactor : RequestParam name 설정
#본문: 코드 의도 설명 가능한 자세하게

# Feat     : 새로운 기능 추가
# Fix      : 버그 수정
# Docs     : 문서 수정
# Test     : 테스트 코드 추가
# Refactor : 코드 리팩토링, 파일 혹은 폴더명을 수정하거나 옮기는 작업만인 경우, 파일을 삭제하는 작업만 수행한 경우
# Design   : CSS 및 레이아웃 작업
# Chore    : 코드 외 빌드 부분 혹은 패키지 매니저 수정사항

zmfpdl64 reviewed (approved):
수고하셨습니다 주원님 👍

Joowon-Seo committed:
refactor : 장바구니 페이징 수정
